src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -375,6 +380,10 @@ export async function zipCode(modulePath: string) {
             zip.addLocalFile(file, path.dirname(paddedPath))
         }
         zipManifest.dependenciesRoot += `${dependencyFolderName}/`
+        telemetry.codeTransform_dependenciesCopied.emit({
+            codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
```

Comment:
https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json
Was this added here?